.env.example

```diff
@@ -1,3 +1,5 @@
 ETHERSCAN_API_KEY=ABC123ABC123ABC123ABC123ABC123ABC1
 ROPSTEN_URL=https://eth-ropsten.alchemyapi.io/v2/<YOUR ALCHEMY KEY>
 PRIVATE_KEY=0xabc123abc123abc123abc123abc123abc123abc123abc123abc123abc123abc1
+RINKEBY_RPC_URL=https://eth-rinkeby.alchemyapi.io/v2/<YOUR-API-LEY>
```

Comment:
Please rename the variable to be same as `ROPSTEN_URL` >>> `RINKEBY_URL`

Reply:
Done